crud/common/call.lua

```diff
@@ -165,4 +135,33 @@ function call.ro(func_name, func_args, opts)
     return call_impl('callro', func_name, func_args, opts)
 end
 
+function call.rw_single(bucket_id, function_name, argument_list, options)
+    local res, err = vshard.router.callrw(bucket_id, function_name, argument_list, options)
+
+    -- This is a workaround, until vshard supports telling us where the error happened
+    if err ~= nil then
+        if type(err) == 'table' and err.type == 'ClientError' and type(err.message) == 'string' then
+            if err.message == string.format("Procedure '%s' is not defined", function_name) then
+               err = NotInitializedError:new("crud isn't initialized on replicaset")
+            end
+        end
+
+
+        local replicaset, err2 = vshard.router.route(bucket_id)
+        if replicaset == nil then
+            return nil, CallError:new("Failed to get replicaset for bucket_id %s: %s", bucket_id, err2)
```

Comment:
Maybe in this case we should return `err` anyway and additionally say that `route` failed and it's not possible to get replicaset UUID?